src/System.Management.Automation/resources/SessionStateStrings.resx

```diff
@@ -281,7 +281,10 @@
   </data>
   <data name="GetContainerContentException" xml:space="preserve">
     <value>Unable to access '{0}' contents as it is a directory. Please use 'Get-ChildItem' instead.</value>
-  </data>  
+  </data>
+  <data name="WriteContainerContentException" xml:space="preserve">
+    <value>Unable to access '{0}' contents as it is a directory.</value>
```

Comment:
I'd prefer `{0}` on end of the string because we support long paths.
```powershell
Unable to write content because it is a directory: {0}
```
For `GetContainerContentException` too.
/cc @SteveL-MSFT

Reply:
Sure, that makes sense.

Reply:
I think the updated error messages are fine, even for `GetContainerContentException`.